schemas/Account.json

```diff
@@ -39,6 +39,10 @@
     "ledger": {
       "description": "A link the the account's ledger",
       "type": "Iri.json"
+    },
+    "fingerprint": {
+      "description": "A fingerprint of the account's client certificate",
+      "type": "string"
```

Comment:
Example of the fingerprint property for a self-signed certificate is `"B7:E7:59:23:FF:E2:2A:9B:5B:12:5C:2C:98:F3:41:F7:FE:80:CC:E7"`

See: https://nodejs.org/api/tls.html#tls_tlssocket_getpeercertificate_detailed

When generating the certificate for testing, I did not specify the `digest` and openssl has this to say

> -digest
> This affects any signing or display option that uses a message digest, such as the -fingerprint, -signkey and -CA options. Any digest supported by the OpenSSL dgst command can be used. If not specified then SHA1 is used. Note that if a DSA key is used for signing, then this flag is ignored and SHA1 is used.

https://www.openssl.org/docs/manmaster/apps/x509.html